components/landingPage.vue

```diff
@@ -0,0 +1,76 @@
+<script>
+  export default {
+    data() {
+      return {
+        url: 'goose.png'
+      }
+    }
+  }
+</script>
+  
+<template>
+    
+    <div class="frontPage">
+      <div> <img :src=url> </div>
```

Comment:
Although this might compile, it's always best to wrap attributes with `"`.

```suggestion
      <div> <img :src="url"> </div>
```